configs/uc-davic.nrel-op.json

```diff
@@ -0,0 +1,83 @@
+                "why_we_collect": "The data collected during the program will be used to understand how e-bikes are used including when and how often as well as distance travelled. This in turn will aid in the estimation of GHG reductio from such incentive programs and new technologies.",
+                "research_questions": ["Travel behavior modeling to support cost-benefit analyses of various potential incentive programs.", "The impact of e-bikes on transportation energy efficiency and GHG emissions"]
+            },
+            "es": {
+                "deployment_partner_name": "Universidad de California Davis",
+                "deployment_name": "Programa de Acceso Comunitario a Bicicletas Eléctricas",
+                "summary_line_1": "El Consejo de Recursos del Aire de California (CARB) apunta a incentivar a personas adoptar modas de transportacion energía eficientes y no contaminantes y maximizar los beneficios del calidad del aire.",
+                "summary_line_2": "En este programa, proporciona bicicletas eléctricas a personas de bajos ingresos en comunidades de todo el estado.",
+                "summary_line_3": "Si es un participante del programa, esta aplicación puede ayudarlo a registrar sus viajes y competir contra otros para beneficiar nuestro medio ambiente.",
+                "short_textual_description": "La creciente popularidad de las bicicletas eléctricas se alinea bien con los objetivos climáticos y de equidad del Estado de Colorado. Las eBikes, aunque su costo todavía está fuera del alcance de algunos, se están volviendo más asequibles a medida que bajan los costos de las baterías y nuevos fabricantes ingresan al mercado. Con la mayoría de los viajes en vehículos motorizados de menos de 10 millas, las bicicletas eléctricas son adecuadas para reemplazar los viajes que normalmente se realizan en un automóvil.",
+                "why_we_collect" : "Los datos recopilados durante el programa se utilizarán para comprender cómo se utilizan las bicicletas eléctricas, incluido cuándo y con qué frecuencia, así como la distancia recorrida.",
+                "research_questions": ["Este programa hará un seguimiento de los viajes en eBike completados por los participantes del programa", "Los proyectos que participen requerirán que los participantes realicen un seguimiento de sus viajes a través de la aplicación."]
+            }
+        }
+    },
+    "survey_info": {
+      "surveys": {
+        "UserProfileSurvey": {
+          "formPath": "json/demo-survey-v2.json",
+          "version": 1,
+          "compatibleWith": 1,
+          "dataKey": "manual/demographic_survey",
+          "labelTemplate": {
+            "en": "Answered",
+            "es": "Contestada"
+          }
+        }
+      },
+      "trip-labels": "MULTILABEL"
+    },
+    "display_config": {
+        "use_imperial": true
+    },
+    "metrics": {
+        "include_test_users": false
```

Comment:
we want to initially include test users; we will flip the switch once they get a reasonable mass of participants